capstone/src/main/java/com/google/univiz/CollegeScoreCardData/CollegeScoreCardInterface.java

```diff
@@ -0,0 +1,17 @@
+import java.util.List;
+
+interface CollegeScoreCardInterface {
```

Comment:
Is this the CollegeScoreCardInterface? Or is this a collection of interfaces that will have a CollegeScoreCard implementation? Considering refactoring this into multiple interfaces:

* CollegeAutocomplete
* MapData
* CollegeData

which in turn can have CollegeScoreCard implementation(s).

Reply:
I was hoping to keep all these functions in the same interface so that only one library will be calling the CollegeScoreCard API. Should I still refactor?

Reply:
Let's consider the autocomplete methods. Right now, this interface is tightly coupled to the entire surface of your API. What happens if you wanted to implement a better autocomplete later (that might have no knowledge of MapData or College Data)? If you had an CollegeAutocomplete interface agnostic of the rest of the signature, then it would be simple for you to wire up a new implementation.